jmeter-service/main.go

```diff
@@ -100,8 +100,7 @@ func runTests(event cloudevents.Event, shkeptncontext string, data keptnv2.TestT
 	var res bool
 	healthCheckWorkload, err = getWorkload(jmeterconf, TestStrategy_HealthCheck)
 	if healthCheckWorkload != nil {
-
-		_ , err := net.LookupIP(serviceUrl.String())
+		err := checkEndpointAvailable(5*time.Second, serviceUrl)
```

Comment:
This introduces an unexpected change of behaviour. Instead of looking up the IP address of the service, we are checking whether the service is reachable via HTTP.